lib/pull.js

```diff
@@ -40,19 +65,19 @@ async function downloadAndSave(projectConfig, token) {
       return;
     }
     if (e.response && e.response.status === 401) {
-      error = "You don't have access to the selected project";
-      msg = `${output.errorText(error)}.\nChoose another using the ${output.info('project')} command, or update your API key.`;
+      error = "You don't have access to the selected projects";
+      msg = `${output.errorText(error)}.\nChoose others using the ${output.info('project')} command, or update your API key.`;
       return console.log(msg);
     }
     if (e.response && e.response.status === 403) {
-      error = "The requested project doesn't have Developer Mode enabled";
-      msg = `${output.errorText(error)}.\nPlease choose a different project using the ${output.info('project')} command, or turn on Developer Mode for this project. Learn more here: ${output.subtle('https://www.dittowords.com/docs/ditto-developer-mode')}.`;
+      error = "One or more of the requested projects don't have Developer Mode enabled";
```

Comment:
These sorts of generic error messages are less than ideal from a UX perspective. There are a couple of approaches that come to mind for making them better:
- Move away from my approach to query a single endpoint for all projects, and instead hit the `/projects/:project_id` endpoint multiple times. Broadly speaking I think this would be less performant on both ends, but would make it really easy to have project-specific error messages instead of "One or more of the request projects..."
- Return project-specific error data from the API to enable displaying more helpful error messages

Reply:
Hm..... that makes sense. I think because we decided on the /projects endpoint for now, we can look into returning more project-specific error data from the API endpoint, but I think in general we can leave a lot of error handling improvements as work for the future